shell/platform/embedder/embedder.h

```diff
@@ -1082,8 +1082,10 @@ typedef struct {
   /// A value that `value` will have after a kFlutterSemanticsActionDecrease`
   /// action has been performed.
   const char* decreased_value;
-  /// The reading direction for `label`, `value`, `hint`, `increasedValue`, and
-  /// `decreasedValue`.
+  /// A textual tooltip attached to the node.
+  const char* tooltip;
+  /// The reading direction for `label`, `value`, `hint`, `increasedValue`,
+  /// `decreasedValue`, and `tooltip`.
```

Comment:
Since we promise ABI stability for the embedder API defined here, this will need to go at the end of the struct. Otherwise, if someone drops in a newly-compiled engine shared library without recompiling their embedder against this code, they'll get the tooltip char* when they try to look up `text_direction`.